build.sbt

```diff
@@ -50,6 +50,7 @@ lazy val rawls = project.in(file("."))
 // This appears to do some magic to configure itself. It consistently fails in some environments
 // unless it is loaded after the settings definitions above.
 Revolver.settings
+Global / excludeLintKeys += debugSettings // To avoid lint warning
```

Comment:
what is the lint warning? Can we fix the source code instead of skipping the linting?

Reply:
The warning's been there since forever, it's from those Revolver settings above, so I don't necessarily want to get into it, since they're apparently magic

Reply:
> I don't necessarily want to get into it

ok, I pulled branches and ran it myself, the lint warning is:
```
[warn] there's a key that's not used by any other settings/tasks:
[warn]  
[warn] * Global / debugSettings
[warn]   +- /Users/davidan/work/src/rawls/build.sbt:52
[warn]  
[warn] note: a setting might still be used by a command; to exclude a key from this `lintUnused` check
[warn] either append it to `Global / excludeLintKeys` or call .withRank(KeyRanks.Invisible) on the key
```
this is present (and fixed by the exclude) on both this PR branch with java 11 and on the develop branch with java 1.8, so it isn't related to the java 11 upgrade as far as I can tell.

Reply:
Ok, so should I roll this line back? I was just trying to get clean console output.